src/app/spartacus/fs-configuration.module.ts

```diff
@@ -64,17 +60,6 @@ import { environment } from '../../environments/environment';
         fallbackLang: 'en',
       },
     }),
-    provideConfig(<IconConfig>{
```

Comment:
This was overriding our own override provided by `icon-config.ts`. This configuration is already being passed in `FSStorefrontModule` in our own lib.